### ISSUE TITLE
How I can get the UseCached file

### ISSUE THREAD

**Issue by jaileon** [datacenter/ucs-browser#1]
When I try to run the script following the instructions on the Readme, doesn't work

`Cache File not found at ~\ucs_cache.ucs`

Could you provide guidelines please

**Comment by techBeck03** [comment]
Follow the instructions in the `Running the Script` section of the readme first and see if that solves the issue.  You must first use the interactive text menu to connect to UCS domains and then cache them before you can use the `-UseCached` flag in future executions.

Thanks!

**Comment by jaileon** [comment]
Thanks @techBeck03, do you know if is possible run it in macOS? 

I tested it with `PowerShell 6.1.1` & `PowerShell 7.0.3` but with none worked

**Comment by techBeck03** [comment]
> Thanks @techBeck03, do you know if is possible run it in macOS?
> 
> I tested it with `PowerShell 6.1.1` & `PowerShell 7.0.3` but with none worked

Yes i've tested this on macOS and it worked just fine.  You have to go through the text based menus first to connect to a domain(s) and cache your credentials then exit the script.  Once your credentials are cached you can run the report using the argument flags described in the readme.

What doesn't work in macOS is creating the report via the interactive text menu because it uses a file prompt that only works in Windows.

Let me know if you still have issues.  Thanks!
